docs/mobile-apps/automated-testing/espresso-xcuitest/real-devices.md

```diff
@@ -139,35 +152,42 @@ You can use any of the existing methods of distribution for your iOS app, except
 
-There are two ways to configure your parameters for testing on Sauce Runner for Real Devices with Espresso and XCUITest: by using a command-line interface or by creating and running a YAML file.
+In Xcode, go to **iOS Deployment Target** and ensure that you've set the same iOS version for both the app and your test runner. If these don't match, your tests will run locally, but fail when you run them against Sauce Labs real devices. From your Xcode **Build Settings**:
 
-### Using the Command Line Interface
-Sauce Runner for Real Devices will execute tests based on the parameters you set in the file.
+1. **Select Your App Project**: Select the Project you want to build, and under **Build Settings**, set the iOS Deployment Target to the iOS operating system version you want to use in your test. All target outputs of this project, including the application and your test runner, will be set to the same operating system version.
+
+2. **Select Your App Target**: Select the **Target** for your Project, and under **Build Settings**, set the **iOS Deployment Target** to the iOS operating system version you want to use in your test. This will also overwrite the Build Settings at the Project level to that operating system version, so if you use this method, be aware that your Targets can become out of sync with each other and the Project settings, and your tests will break. If you change the iOS version for one target output, you may want to build the Project again to make sure all your targets are in sync.
 
-#### Sauce Runner RDC CLI Reference
+
+## Test Configuration Options
+
+There are two ways to configure your parameters for testing on Sauce Runner for Real Devices with Espresso and XCUITest: via CLI or YAML file. Sauce Runner for Real Devices will execute tests based on the parameters you se.
+
+### Using the Command Line Interface
 
 Add one of the following commands to your test script:
   * `xcuitest` Defines XCUITest as the test framework to use for your native iOS app tests
   * `espresso` Defines Espresso as the test framework to use for your native Android app tests
 
-From here, head to the full list of commands and options at [Sauce Runner for Real Devices Command-Line Reference](dev/cli/espresso-xcuitest.md).
+From here, head to the full list of commands and options at [Sauce Runner for Real Devices CLI Reference](dev/cli/espresso-xcuitest.md).
 
-### Using a YAML Configuration File
+### Using YAML Configuration File
 
-As an alternative to configuring your [Espresso and XCUITest RDC tests](mobile-apps/automated-testing/espresso-xcuitest/real-devices.md) using the [command line interface](/dev/cli/espresso-xcuitest.md), you can create a YAML configuration file.
+As an alternative to configuring your Espresso and XCUITest RDC tests using the [command line interface](/dev/cli/espresso-xcuitest.md), you can create and run a YAML configuration file.
```

Comment:
The yml example is still with the `apiKey`

Reply:
Good catch